rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.yml

```diff
@@ -2,13 +2,12 @@ AnalysisType: rule
 Filename: cloudflare_httpreq_bot_high_volume_greynoise.py
 RuleID: Cloudflare.HttpRequest.BotHighVolumeGreyNoise
 DisplayName: Cloudflare Bot High Volume GreyNoise
-Enabled: false
+Enabled: true
```

Comment:
action needed: 
this has the same logical bug as the other httpreq_bot_high_volume.. also, there is substantial overlap between both detections and their greynoise variants. could we consolidate into one detection( presumably the greynoise version )

Reply:
Flipping this to false for now while we update this detection in another branch.